src/main/kotlin/it/pagopa/ecommerce/eventdispatcher/services/v2/AuthorizationStateRetrieverService.kt

```diff
@@ -45,4 +54,55 @@ class AuthorizationStateRetrieverService(
         }
       })
   }
+
+  fun getOrder(
```

Comment:
add some logs here, for the switchIfEmpty f.e. (get order called for invalid payment gateway), the parameters used for the get order NPG api call, such as transactionId, correlationId, pspId, paymentMethodId and so on

Reply:
Solved by https://github.com/pagopa/pagopa-ecommerce-event-dispatcher-service/pull/269/commits/27e57497cfc8838b76b0ca687ced8f1f2856a52f